linux_os/guide/system/software/integrity/fips/grub2_enable_fips_mode/bash/shared.sh

```diff
@@ -1,4 +1,4 @@
-# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,Red Hat Enterprise Linux 8,multi_platform_fedora
+# platform = Red Hat Enterprise Linux 7
```

Comment:
Still applicable to rhel8. Please undo the changes.

Reply:
@redhatrises I don't know if we should check `/etc/default/grub` on RHEL8 because the utility `fips-mode-setup` which should be used for enabling FIPS mode on RHEL8 doesn't modify that `/etc/default/grub`. On the other hand, we could say that running `fips-mode-setup`  is not enough and users are required to modify `/etc/default/grub` as well.

Does anybody have any idea if modifying  `/etc/default/grub` should still be required in order to enable FIPS mode in RHEL8?

Reply:
Running fips-mode-setup is enough.

Reply:
@jan-cerny @t8m what happens on a kernel upgrade? Also what happens if someone sets `fips=0` in `/etc/default/grub`

Reply:
Kernel upgrade should keep the kernel options as edited by grubby. Unfortunately there is a regression in this with the new grubby wrapper for BLS which needs to be fixed. I am not sure what happens if users set conflicting options via grubby and via /etc/default/grub however please note that for example there is no grub on s390x so I do not think it makes much sense to look at /etc/default/grub as it is not a canonical source of the information.

Reply:
While that may be true, we will need to validate that fips hasn't been disabled elsewhere. So using /etc/default/grub as an example, if an administrator can set `fips=0` in /etc/default/grub which overrides grubby, that's a compliance issue that we need to flag with an OVAL check.